src/docs/Views.mdx

```diff
@@ -0,0 +1,249 @@
+        paragraph.textContent = 'This extension offers the following functionalities: ';
+        messageNode.appendChild(paragraph);
+
+        const features:string[] = ['Hello','World','Theia','!'];
+
+        const featureInfoContent = document.createElement('ol');
+        messageNode.appendChild(featureInfoContent);
+
+        features.forEach(feature => {
+            const featureInfo = document.createElement('li');
+            featureInfo.textContent = feature;
+            featureInfoContent.appendChild(featureInfo);
+        });
+
+        this.appendAcceptButton('Close');
+    }
+
+    get value(): undefined { return undefined; }
+}
+```
+
+Afterwards, create a constant to store the information about a new command to open our dialog in your extension-name-contribution.ts
+
+```typescript
+export const OpenDialog = {
+    id: 'OpenDialog',
+    label: "Open Dialog"
+};
+```
+
+Inject a dialog instance in your CommandContribution class.
+
+```typescript
+@injectable()
+export class YourExtensionCommandContribution implements CommandContribution {
+
+    constructor(
+        @inject(Dialog) protected readonly dialog: Dialog,
+    ) { }
+
+}
+```
+Register the command which opens the dialog.
+
+```typescript
+registerCommands(registry: CommandRegistry): void {
+ registry.registerCommand(OpenDialog, {
+            execute: () => this.openDialog()
+        });
+}
+```
+
+We also need to add a menu contribution so we can easily open our dialog in the "Help" menu path.
+
+```typescript
+@injectable()
+export class YourExtensionMenuContribution implements MenuContribution {
+
+    registerMenus(menus: MenuModelRegistry): void {
+        menus.registerMenuAction(CommonMenus.HELP, {
+            commandId: OpenDialog.id
+        });
+```
+
+## Using Theia styles
+Running this will work but actually it won't look pleasing. Our text is black and therefore hard to read with the dark theme. Theia is making use of CSS variables to define common colors, fonts and styles. The dark and light theme respectively declare these variables. So by using these common variables, we ensure that our UI looks good in both the dark and light theme (and possible further themes which declare these variables). There are two ways of adding CSS to our DOM elements: either by directly manipulating the DOM or by using a custom stylesheet. We'll use the first approach in this example and the latter one is being used in the view example.
+
+Let's add the base font and color to our root div element:
+```typescript
+    messageNode.setAttribute('style', 'font-family: var(--theia-ui-font-family); color: var(--theia-ui-font-color1);');
+```
+Now our Dialog is readable and we could further enhance it's appearance by adding custom CSS.
+
+For reference, you can find all the variables in [variables-dark.useable.css](https://github.com/theia-ide/theia/blob/master/packages/core/src/browser/style/variables-dark.useable.css) (or the respective css of the light theme).
```

Comment:
The link to the main Theia repo is outdated.
https://github.com/eclipse-theia/theia/blob/master/packages/core/src/browser/style/variables-dark.useable.css